prime-router/build.gradle.kts

```diff
@@ -75,6 +75,13 @@ val reportsApiEndpointHost = (
 val jooqSourceDir = "build/generated-src/jooq/src/main/java"
 val jooqPackageName = "gov.cdc.prime.router.azure.db"
 
+fun addVaultValuesToEnv(env: MutableMap<String, Any>) {
```

Comment:
Can you add some comments on what this does?

Reply:
Done